tests/unit/s2n_bike_r1_kat_test.c

```diff
@@ -26,11 +26,17 @@
 
 int main(int argc, char **argv, char **envp) {
     BEGIN_TEST();
+
+#if !S2N_NO_PQ
```

Comment:
Typically you want to use 

#if !defined() for this stuff

Or #ifndef

Reply:
Fixed